doc/source/serve/deployment.rst

```diff
@@ -206,6 +206,102 @@ Please refer to the Kubernetes documentation for more information.
 Monitoring
 ==========
 
+Logging
+-------
+
+.. note::
+
+  For an general overview of logging in Ray, see `Ray Logging <../ray-logging.html>`__.
+
+When looking through log files of your Ray Serve application, it is useful to know which backend and replica each log line originated from.
+To automatically tag your logs with the current backend tag and replica tag (of the form ``backend_tag#<random letters>``), use the following function:
+
+.. autofunction:: ray.serve.get_backend_logger
+
+To write your own custom logger using Python's ``logging`` package, you may find the following two methods useful:
+
+.. autofunction:: ray.serve.get_current_backend_tag
+.. autofunction:: ray.serve.get_current_replica_tag
+
+Ray Serve logs can be ingested by your favorite external logging agent.  Ray logs from the current session are exported to the directory `/tmp/ray/session_latest/logs` and remain there until the next session starts.
+
+Here is a quick walkthrough of how to explore and filter your logs using `Loki <https://grafana.com/oss/loki/>`__.  
```

Comment:
Note here that we'll set it up manually but it's much easier to configure on kubernetes. Actually, it might be worth having another tutorial on k8s. We can do that in the future.